tests/robustness/report/wal.go

```diff
@@ -199,6 +199,13 @@ func parseEntryNormal(ent raftpb.Entry) (*model.EtcdRequest, error) {
 			OperationsOnFailure: []model.EtcdOperation{},
 		}
 		for _, cmp := range raftReq.Txn.Compare {
+			if ver := cmp.GetVersion(); ver > 0 {
```

Comment:
Can you create a switch based on cmp.Target and cmp.Result?

Reply:
Done